pages/about.html

```diff
@@ -4,9 +4,43 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
```

Comment:
The links to styles, pages, and images work in the live server but not when opened statically in the browser. Not an issue, but be careful if you're deploying that relative pathing might need to be used!